File: Code/Second_work/Задание 2.pas
var
  a, b, choice: integer;
  result: real;
begin
  writeln('Введите первое число:');
  readln(a);
  writeln('Введите второе число:');
  readln(b);
  writeln('Выберите операцию:');
  writeln('1 — сложение');
  writeln('2 — вычитание');
  writeln('3 — умножение');
  writeln('4 — деление');
  readln(choice);
  case choice of
    1: begin
         result := a + b;
         writeln('Результат: ', result);
       end;
    2: begin
         result := a - b;
         writeln('Результат: ', result);
       end;
    3: begin
         result := a * b;
         writeln('Результат: ', result);
       end;
    4: if b = 0 then
         writeln('Ошибка: деление на ноль!')
       else begin
         result := a / b;
         writeln('Результат: ', result);
       end;
    else
      writeln('Неверный выбор операции.');
  end;
end.

File: Code/Second_work/Задание 2.py
a = float(input('Введите первое число: '))
b = float(input('Введите второе число: '))
print('Выберите операцию:')
print('1 — сложение')
print('2 — вычитание')
print('3 — умножение')
print('4 — деление')
choice = input('Ваш выбор (1/2/3/4): ')
if choice == '1':
    result = a + b
    print(f'Результат: {result}')
elif choice == '2':
    result = a - b
    print(f'Результат: {result}')
elif choice == '3':
    result = a * b
    print(f'Результат: {result}')
elif choice == '4':
    if b == 0:
        print('Ошибка: деление на ноль!')
    else:
        result = a / b
        print(f'Результат: {result}')
else:
    print('Неверный выбор операции.')
